dapp/src/components/page/proposal/ExecuteProposalModal.tsx

```diff
@@ -0,0 +1,99 @@
+import React from "react";
+import { useState, useEffect } from "react";
+import JsonView from "react18-json-view";
+import { modifySlashInXdr, processDecodedData } from "utils/utils";
+import { stellarLabViewXdrLink } from "constants/serviceLinks";
+import { demoOutcomeData } from "constants/demoProposalData";
+
+interface VotersModalProps {
+  xdr: string;
+  onClose: () => void;
+}
+
+const VotersModal: React.FC<VotersModalProps> = ({ xdr, onClose }) => {
+  const [content, setContent] = useState<any>(null);
+
+  const signAndExecute = () => {};
+
+  const getContentFromXdr = async (_xdr: string) => {
+    try {
+      if (_xdr) {
+        const decoded = processDecodedData(_xdr);
+        console.log("decode:", decoded);
+        setContent(demoOutcomeData);
+      }
+    } catch (error) {
+      console.error("Error decoding XDR:", error);
+    }
+  };
+
+  useEffect(() => {
+    getContentFromXdr(xdr);
+  }, [xdr]);
+
+  return (
+    <div
+      className="fixed inset-0 bg-black bg-opacity-50 flex justify-center items-center z-[9999]"
+      onClick={(e) => {
+        if (e.target === e.currentTarget) {
+          onClose();
+        }
+      }}
+    >
+      <div
+        className="modal relative w-80 sm:w-max sm:max-w-[90%] md:max-w-[620px] px-3 sm:px-5 md:px-9 py-2 sm:py-6 md:py-12 rounded-lg sm:rounded-xl md:rounded-[20px] bg-white my-4 max-h-[90vh] overflow-y-auto"
+        onClick={(e) => e.stopPropagation()}
+      >
+        <div className="flex flex-col gap-1 sm:gap-2">
+          <div className="text-sm sm:text-lg md:text-[22px]">Outcome</div>
+          <div className="flex flex-col gap-1 sm:gap-2 md:gap-3">
+            <a
+              href={`${stellarLabViewXdrLink}${modifySlashInXdr(xdr)};;`}
+              target="_blank"
+              rel="noreferrer"
+              className="flex items-center gap-0.5 sm:gap-1 text-black hover:text-blue  group"
+            >
+              <svg
+                role="img"
+                viewBox="0 0 24 24"
+                xmlns="http://www.w3.org/2000/svg"
+                id="Stellar--Streamline-Simple-Icons.svg"
+                height="24"
+                width="24"
+                fill="#000000"
+                className="group-hover:fill-blue w-3 h-3 sm:w-4 sm:h-4 md:w-5 md:h-5"
+              >
+                <desc>Stellar Streamline Icon: https://streamlinehq.com</desc>
```

Comment:
Is it required by their license that we have their url? I am not a fan to be honest.